cli/pkg/schema/schema_test.go

```diff
@@ -481,9 +481,13 @@ entrypoints:
       /api/:
         name: user-api
 databases:
-  user-db: {}
+  user-db:
+    env_var_key: USER_DB_URL
 websites:
-  public-site: {}
+  public-site:
+    base_dir: ./public
+    asset_dir: ./assets
+    error_page: 404.html
```

Comment:
I didn't think that websites were supported?

Reply:
We have those properties in schema.json marked as required, do we need to remove them then?

E.g.

```json
    "WebsiteIntent": {
      "properties": {
        "baseDir": {
          "type": "string"
        },
        "assetDir": {
          "type": "string"
        },
        "errorPage": {
          "type": "string"
        },
        "build": {
          "type": "string"
        },
        "dev": {
          "$ref": "#/$defs/WebsiteDev"
        }
      },
      "additionalProperties": false,
      "type": "object",
      "required": ["baseDir", "assetDir", "errorPage"]
    }
```